pkg/kn/commands/configuration_edit_flags.go

```diff
@@ -55,5 +75,47 @@ func (p *ConfigurationEditFlags) Apply(config *servingv1alpha1.ConfigurationSpec
 	if err != nil {
 		return err
 	}
+	limitsResources, err := p.computeLimitsResources()
+	if err != nil {
+		return err
+	}
+	requestsResources, err := p.computeRequestsResources()
+	if err != nil {
+		return err
+	}
+	err = servinglib.UpdateResources(config, requestsResources, limitsResources)
+	if err != nil {
+		return err
+	}
 	return nil
 }
+
+func (p *ConfigurationEditFlags) computeRequestsResources() (corev1.ResourceList, error) {
+	requestsCPU, err := resource.ParseQuantity(p.RequestsFlags.CPU)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	requestsMemory, err := resource.ParseQuantity(p.RequestsFlags.Memory)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	return corev1.ResourceList{
+		corev1.ResourceCPU:    requestsCPU,
+		corev1.ResourceMemory: requestsMemory,
+	}, nil
+}
+
+func (p *ConfigurationEditFlags) computeLimitsResources() (corev1.ResourceList, error) {
+	limitsCPU, err := resource.ParseQuantity(p.LimitsFlags.CPU)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	limitsMemory, err := resource.ParseQuantity(p.LimitsFlags.Memory)
```

Comment:
Wouldn't leaving out the flag have p.LimitsFlags.Memory be "" and thus make ParseQuantity return the zero Quantity and an error? At the very least a comment about this case is warranted.

Reply:
Yeah, my bad again. Will fix that. Rushing a bit here. Will fix

Reply:
addressed in above if I understood your comment. See previous.

Reply:
Already addressed.